conf4j-core/src/main/java/com/sabre/oss/conf4j/converter/DefaultTypeConverters.java

```diff
@@ -59,6 +74,8 @@ private DefaultTypeConverters() {
      */
     private static TypeConverter<Object> prepareDefaultTypeConverter() {
         TypeConverter<Object> simpleTypeConverters = new ChainedTypeConverter(asList(
+                new LocalDateTimeTypeConverter(),
```

Comment:
I think it would be better to move these new converter at the end of list. _LocalDateTimeTypeConverter_ can be moved just after _DurationTypeConverter_, _UrlTypeConverter_ just after _PatternTypeConverter_.